java/README.md

```diff
@@ -29,27 +30,49 @@ Software Dependencies:
 - protoc (protobuf compiler)
 - Rust
 
+Please also consider installing the following packages to build [GLIDE core rust library](../glide-core/README.md):
+
+- GCC
+- pkg-config
+- openssl
+- openssl-dev
+
 #### Prerequisites
 
+**Protoc installation**
+
+Download a binary matching your system from the [official release page](https://github.com/protocolbuffers/protobuf/releases/tag/v25.1) and make it accessible in your $PATH by moving it or creating a symlink.
```

Comment:
```suggestion
Download a binary matching your system from the [official release page](https://github.com/protocolbuffers/protobuf/releases) and make it accessible in your $PATH by moving it or creating a symlink.
```
Note: we only support latest.  And really, we want to only support https://github.com/protocolbuffers/protobuf/releases/tag/v26.0-rc2 and later because of https://github.com/protocolbuffers/protobuf/issues/12639

Reply:
I put here the same version we use in CI